spec/outcome_checker_spec.rb

```diff
@@ -0,0 +1,132 @@
+require 'outcome_checker'
+require 'pry'
+
+class TestBoardForOutcomeChecker
+  attr_accessor :test_lines, :test_full
+
+  def initialize(test_lines:, test_full:)
+    @test_lines = test_lines
+    @test_full = test_full
+  end
+
+  def get_space(row, column)
+    index = (row * 3) + column
+    values[index]
+  end
+
+  def mark_space(token, space); end
+
+  def lines
+    test_lines
+  end
+
+  def full?
+    test_full
+  end
+
+  def available?; end
+end
+
+def test_lines_win_with_full_board
+  rows = [ ['X', 'O', 'X'], ['O', 'X', 'O'], ['X', 'X', 'O'] ]
+  columns = rows.transpose
+  diagonals = [ ['X', 'X', 'O'], ['X', 'X', 'X']]
+
+  rows + columns + diagonals
+end
+
+def test_lines_win_with_open_spaces
+  rows = [ ['X', 'O', 'X'], ['O', 'X', 'O'], ['X', 8, 9]]
```

Comment:
To test this class we don't need an accurate number of `X` and `O`, as that is the responsibility of where the turns are managed. I would clear this test lines and leave only those marks relevant to the test so that they are easier to read. e.g.:

```ruby
rows = [
  [  1,   2, 'X'],
  [  4, 'X',   6],
  ['X',   8,   9]
]
```

Reply:
Game loop is currently just checking to see if there is a win or a draw, not whose turn it was on a win. I'll take a look at that in the next ticket when I work on displaying the results. 

Thanks for the suggestion about making the tests more readable, and catching the "3 nils in a row" case! I'll take a stab at refactoring those parts.